### PR TITLE
Include deadly skills list in introduction. (again)

### PR DESCRIPTION
**Pull request by mTrilby**
Pull request to include my deadly skills in the master repo. (updated comment)

### CODE REVIEW

**Comment by mTrilby** [comment]
This is an additional comment to pass the time while github allows contributer access to content.

**Comment by mTrilby** [comment]
Still having issues confirming my pull request submission.
Hmmm....   why??

**Comment by mTrilby** [comment]
I think we can accept this. :+1:

File: introduction.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 Introduction MD file contents.
 Updates to the MD file.
-Another update to the file.
-My favorite CodeSchool path is JavaScript although I have not started on it yet.
+List of my deadly skills:
+* Git
+* JavaScript
+* CSS & HTML
+
+Still having issues with the validation of my pull request.
```
Comment (codeschool-kiddo) [inline introduction.md]: Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

**Pull request merged**